stock_available/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2014 Numérigraphe
+# Copyright 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
```

Comment:
Could you please add some missing entries like in https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L9